Application/src/model/Menu.java

```diff
@@ -287,24 +447,53 @@ public void identifyState() {
             pause();
         }
 
+        Platform.runLater(() -> {
```

Comment:
This is some really amazing code! How did you come to this conclusion that it would work?

Reply:
I asked a magician that lives on the fourth floor in Patricia. He directed me to a hidden expert that showed me the way to enlightenment. Then I asked Pontus about the buttons and he told me to add it there.